docs/api-markdown-documenter/build-api-nav.js

```diff
@@ -0,0 +1,76 @@
+const { ApiItemKind, ApiItemUtilities } = require("@fluid-tools/api-markdown-documenter");
+
+const fs = require("fs-extra");
+const path = require("path");
+const yaml = require("js-yaml");
+
+/**
+ * Processes documents and generates data required for the nav bar.
+ * @param {Array<Object>} documents - List of documents with apiItem.
+ * @param {ApiItem | undefined} documents.apiItem - The API item that the document is created from. Some documents may not have an apiItem.
+ */
+async function buildNavBar(documents) {
+	const validKinds = new Set([
```

Comment:
Nit: How about `navKinds` - "valid" (at least to me) would indicate that other kinds would result in failure of some kind.

Reply:
Actually, it looks like the only part of the `Document` items we care about are their API items. Would it make sense for this function to take in an array of ApiItem_s instead? We could then call it with `documents.map(document => document.apiItem).filter(apiItem => apiItem !== undefined)` (or something).

Total nitpicks, feel free to disregard :)